apispec/insolvency-delta-spec.yml

```diff
@@ -0,0 +1,158 @@
+post:
+  summary: Accepts an incoming insolvency delta, transforms it into an avro schema and puts it onto a Kafka topic.
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: '#/components/schemas/Insolvency'
+    required: true
+  responses:
+    "200":
+      description: Successfully produced message onto Kafka topic.
+    "400":
+      description: Bad request body - validation errors.
+    "401":
+      description: Unauthorised - missing api key in header.
+    "500":
+      description: Internal server error has occured.
+components:
+  schemas:
+    Insolvency:
```

Comment:
I think the mappings may have a missing delta_at, might need to add the following

```
Insolvency_delta:
      type: object
      properties:
        insolvency:
          type: object
          items:
            $ref: '#/components/schemas/Insolvency'
        delta_at:
          type: string
      required:
        - insolvency
        - delta_at
```

Reply:
Can't see it in the test data file here https://companieshouse.atlassian.net/wiki/spaces/DSPR/pages/3616276522/Test+Data nor in the excel spreadsheet

Reply:
Let me confirm this

Reply:
delta_at needs to be added by the DBAs

Reply:
Sorry - just checked, delta_at is already added in Chips, but it is in insolvency, not outside, e.g. 
{
"insolvency":[
{"company_number":"05489886"
**,"delta_at": "20211008152823383176"**
,
"case_numbers":[
	{
"case_number":"1",
"case_type_id":"13",

Reply:
Alright we will add it as a mandatory field